demoScripts/aksQuickStart.md

```diff
@@ -0,0 +1,426 @@
+#Customer intent: As a developer or cluster operator, I want to quickly create an AKS cluster and deploy an application so that I can see how to run and monitor applications using the managed Kubernetes service in Azure.
+---
+
+# Quickstart: Deploy an Azure Kubernetes Service cluster using the Azure CLI
+
+Azure Kubernetes Service (AKS) is a managed Kubernetes service that lets you quickly deploy and manage clusters. In this quickstart, you will:
+
+* Deploy an AKS cluster using the Azure CLI.
+* Run a sample multi-container application with a web front-end and a Redis instance in the cluster.
+
+:::image type="content" source="media/quick-kubernetes-deploy-portal/azure-voting-application.png" alt-text="Screenshot of browsing to Azure Vote sample application.":::
+
+This quickstart assumes a basic understanding of Kubernetes concepts. For more information, see [Kubernetes core concepts for Azure Kubernetes Service (AKS)][kubernetes-concepts].
+
+[!INCLUDE [quickstarts-free-trial-note](../../../includes/quickstarts-free-trial-note.md)]
+
+To learn more about creating a Windows Server node pool, see [Create an AKS cluster that supports Windows Server containers](quick-windows-container-deploy-cli.md).
+
+[!INCLUDE [azure-cli-prepare-your-environment.md](~/articles/reusable-content/azure-cli/azure-cli-prepare-your-environment.md)]
+
+- This article requires version 2.0.64 or later of the Azure CLI. If using Azure Cloud Shell, the latest version is already installed.
+
+- The identity you are using to create your cluster has the appropriate minimum permissions. For more details on access and identity for AKS, see [Access and identity options for Azure Kubernetes Service (AKS)][aks-identity-concepts].
+
+- If you have multiple Azure subscriptions, select the appropriate subscription ID in which the resources should be billed using the
+[az account][az-account] command.
+
+- Verify *Microsoft.OperationsManagement* and *Microsoft.OperationalInsights* providers are registered on your subscription. These are Azure resource providers required to support [Container insights][azure-monitor-containers]. To check the registration status, run the following commands:
+
+    ```azurecli
+    az provider show -n Microsoft.OperationsManagement -o table
+    az provider show -n Microsoft.OperationalInsights -o table
+    ```
+
+    If they are not registered, register *Microsoft.OperationsManagement* and *Microsoft.OperationalInsights* using the following commands:
+
+    ```azurecli
+    az provider register --namespace Microsoft.OperationsManagement
+    az provider register --namespace Microsoft.OperationalInsights
+    ```
+
+> [!NOTE]
+> Run the commands with administrative privileges if you plan to run the commands in this quickstart locally instead of in Azure Cloud Shell.
+
+## Define Environment Variables
+```azurecli-interactive
+export RESOURCE_GROUP_NAME=myResourceGroup4
+export RESOURCE_LOCATION=eastus
+export AKS_CLUSTER_NAME=myAKSCluster
```

Comment:
I think it makes sense to make naming variables less generic. Should we also put a random identifier at the end in case someone wants to run the tutorial multiple times?

Reply:
For readability I would avoid a random postfix. In other reviews I've said we need to explain why this content is here. So perhaps we should have a boilerplace that includes the reasoning and gives an example of how to add a random postfix if desired.